components/x-gift-article/src/ReferAFriend.jsx

```diff
@@ -12,13 +12,14 @@ export default ({ rafTitle, rafDescription, urls, actions }) => {
 				data-trackable={UrlType.raf + 'Link'}
 			>
 				<span className="o-forms-input o-forms-input--text">
+					<label for="refer-friend-link">{rafTitle}</label>
```

Comment:
This will add the same text twice, isn't it ? I think this is not the final solution due as i understand the view has changed and the content of rafTitle is shown twice.

Reply:
maybe in this case this is not an accessibility issue and we need to discuss with they